config/routes.rb

```diff
@@ -1,9 +1,13 @@
 Rails.application.routes.draw do
   root to: 'taggings#lookup'
 
+  get '/taxons/trash', to: 'taxons#trash', as: 'taxons_trash'
+
   resources :taxons do
     get :confirm_delete
+    get :restore
```

Comment:
I think you can do something like `get :trash, on: :collection` for the `trash` method.